components/solutions/comparisonPage.tsx

```diff
@@ -75,10 +75,7 @@ export default function ComparisonPage(comp: ComparisonInfo) {
           <br />
         </Container>
       </div>
-      <FeatureBanner
-        category="Customer Data Platform"
-        features={comp.features}
-      />
+      <FeatureBanner category="Data Tool" features={comp.features} />
```

Comment:
This is used for all three comparisons (Segment, HighTouch, and Census) so it didn't completely make sense to reference Reverse ETL as not all of the competitors are.

Reply:
Hm, i think I'm ok with using "Reverse ETL" in this case to get the words on the page. Segment technically does reverse ETL in addition to various other things, so it's not wrong.

Reply:
Oh great! Will make that change.